gatsby/plugins/gatsby-source-bnf/src/node-creation/interactions.ts

```diff
@@ -13,10 +14,16 @@ export type InteractantNodeInput = FeedInteractant & {
 	id: string;
 };
 
+export type InteractionNodeInput = FeedInteraction & {
+	id: string;
+};
+
 export const createInteractionNodes = (
 	{ introduction, interactants, messages }: FeedInteractions,
 	sourceNodesArgs: SourceNodesArgs
 ): void => {
+	createBnfNode(introduction, BnfNode.SimpleRecord, sourceNodesArgs);
```

Comment:
I think we need a new node type for InteractionsIntroduction here so we can query it in the introduction page
```suggestion
	createBnfNode(introduction, BnfNode.InteractionsIntroduction, sourceNodesArgs);
```